### PR TITLE
Unstable

### PR DESCRIPTION
**Pull request by TheQuaccer**
Quacc was forced, Quacc wants out of Alzii basement.

### CODE REVIEW

**Review by Alziibun — CHANGES_REQUESTED** [review]
remove the `PERD` from all the book item names.

**Review by TheQuaccer — COMMENTED** [review]
Test1

File: Contents/mods/Perdition/media/scripts/Perdition_CombatSkillBooks.txt [hunk 1]
```diff
@@ -0,0 +1,942 @@
...
+        NumberOfPages       =   220,
+        Weight              =   0.8,
+        Type                =   Literature,
+        DisplayName         =   Spear Vol. 1,
+        Icon                =   Book,
+        SkillTrained        =   Spear,
+        LvlSkillTrained     =   1,
+        NumLevelsTrained    =   2,
+        StaticModel         =   Book,
+        WorldStaticModel    =   BookClosedGround,
```
Comment (Alziibun) [inline Contents/mods/Perdition/media/scripts/Perdition_CombatSkillBooks.txt:561]:
```suggestion
        WorldStaticModel    =   BookClosedGround,
        ReplaceOnUse = Spear1
```

File: Contents/mods/Perdition/media/scripts/Perdition_CombatSkillBooks.txt [hunk 2]
```diff
@@ -0,0 +1,942 @@
...
+        NumberOfPages       =   380,
+        Weight              =   0.8,
+        Type                =   Literature,
+        DisplayName         =   Axe Vol. 5,
+        Icon                =   Book,
+        SkillTrained        =   Axe,
+        LvlSkillTrained     =   9,
+        NumLevelsTrained    =   2,
+        StaticModel         =   Book,
+        WorldStaticModel    =   BookClosedGround,
```
Comment (Alziibun) [inline Contents/mods/Perdition/media/scripts/Perdition_CombatSkillBooks.txt:541]:
```suggestion
        WorldStaticModel    =   BookClosedGround,
        ReplaceOnUse = Axe5,
```

**Review by Alziibun — APPROVED** [review]

**Pull request merged**